doc/rtd/reference/examples.rst

```diff
@@ -117,7 +117,7 @@ Adjust mount points mounted
    :language: yaml
    :linenos:
 
-``Configure instance's SSH keys``
+Configure instance's SSH keys
 =================================
```

Comment:
Thanks for the fix @crd477. When changing section title line length please also reduce this number of == to match the reduced line length. I'll make that change on this PR and merge away. thanks again.